client/src/components/navbar/Navbar.js

```diff
@@ -0,0 +1,90 @@
+import React, { useContext } from "react";
+import {
+  AppBar,
+  Toolbar,
+  IconButton,
+  Button,
+  Box,
+  Hidden,
+} from "@material-ui/core";
+import { makeStyles } from "@material-ui/core";
+import MenuIcon from "@material-ui/icons/Menu";
+import { Link } from "react-router-dom";
+import MobileNavbar from "./mobileNavbar";
+import { UserContext } from "../../context/Context";
+import logo from "../../img/logo.png";
+
+const useStyles = makeStyles(theme => ({
+  root: { flexgrow: 1 },
+  menuButton: { marginRight: "auto", color: "red" },
+  toolbar: {
+    backgroundColor: "transparent",
+    boxShadow: "0 0 0",
+  },
+  toolbarAuth: {
+    backgroundColor: "white",
+  },
+  sitterLink: { color: "black", marginRight: "30px" },
```

Comment:
You should be able to get rid of the underline for this by adding textDecoration: "none" to closer match the mocks

Reply:
But this one is according to mocks isnt it?

Reply:
Oops, you're right! That's my bad